### PR TITLE
Assessment - Add features

### PR DESCRIPTION
**Pull request by ongsterr**
Hi Mike. Please review the pull request for adding status filter for job cards and form live price update feature.

### Done Checklist

* [x] Task has been completed as per the Trello card
* [x] All acceptance criteria are met
* [x] Developer has tested this locally
* [x] Developer has reviewed their own PR
* [ ] Design has been reviewed by design team
* [ ] Tests have been added or updated
* [x] Views are responsive for mobile devices
* [x] Feature files have been written or updated
* [x] Changelog updated

### CODE REVIEW

File: meteor-app/imports/ui/assessment/assessment-add.js [hunk 1]
```diff
@@ -203,6 +209,11 @@ class AssessmentAdd extends Component {
         services: minorServices,
         package: "Minor Service Package"
       },
+      costData: {
+        serviceCost: 6000,
```
Comment (mikkelking) [inline meteor-app/imports/ui/assessment/assessment-add.js]: Prices should be in the /imports/api/constants.js file

Ideally they should be in the database, but I will allow just moving it to the constants file for now

File: meteor-app/imports/ui/assessment/assessment-add.js [hunk 2]
```diff
@@ -223,6 +234,11 @@ class AssessmentAdd extends Component {
         services: majorServices,
         package: "Major Service Package"
       },
+      costData: {
+        serviceCost: 12000,
```
Comment (mikkelking) [inline meteor-app/imports/ui/assessment/assessment-add.js]: As above

**Comment by ongsterr** [comment]
Thanks Mike. I really need to iron this "no constants in components" mindset in my head. I have update it to pull from database.

**Comment by ongsterr** [comment]
Hey Mike. Just pause on my pull request. I need to add one more change :)

**Comment by ongsterr** [comment]
Hi Mike. All changes should be in. Merge conflict should be resolved too. Please review. Thanks!

**Pull request merged**